toon.py

```diff
@@ -12,31 +12,49 @@
 import homeassistant.helpers.config_validation as cv
 
 # Home Assistant depends on 3rd party packages for API specific code.
-REQUIREMENTS = ['toonlib==0.4.0']
+REQUIREMENTS = ['toonlib==0.5.0']
 
 _LOGGER = logging.getLogger(__name__)
 
 DOMAIN = 'toon'
 TOON_HANDLE = 'toon_handle'
+CONF_GAS = 'gas'
+DEFAULT_GAS = True
+CONF_SOLAR = 'solar'
+DEFAULT_SOLAR = False
```

Comment:
Hm... what about calling the update() method on the instantiation of the datastore, like two times to brute force it a bit? The problem is that we need to make sure that the initial state is identified correctly by the component since as you say it cannot be recovered. So lets brute force the init to try to make as sure as possible that we got everything. What do you think?